src/components/ui/formik/RadioField.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { PropsWithoutRef } from 'react';
+import { FieldAttributes, useField } from 'formik';
+import { Radio, RadioProps } from '@patternfly/react-core';
+
+type RadioFieldProps = PropsWithoutRef<FieldAttributes<RadioProps>>;
+
+export const RadioField = (props: RadioFieldProps) => {
+  const [, , { setValue }] = useField({
+    name: props.name,
+    type: 'radio',
```

Comment:
I had to remove this to get the 'field.value' working as expected.

Alternatively, and if we wanted to make the RadioField generic, we would have to let formik use it as radio which requires to let it set value as string which requires us to create a custom form value and derive the userManagedNetworking boolean from it in handleSubmit.